addon/serializers/json.js

```diff
@@ -1535,4 +1538,40 @@ if (isEnabled("ds-payload-type-hooks")) {
 
 }
 
+if (isEnabled("ds-serialize-id")) {
+
+  JSONSerializer.reopen({
+
+    /**
+     serializeId can be used to customize how id is serialized
+
```

Comment:
I think it would be good to outline the default implementation. Something along the lines of:

> By default the snapshots' `id`(which is a String) will be set on the json hash via `json[primaryKey] = snapshot.id`.